### PR TITLE
Update string representation of color spaces

### PR DESCRIPTION
**Pull request by iamcarbon**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/SixLabors/ImageSharp/pulls) open
- [x] I have verified that I am following matches the existing coding patterns and practice as demonstrated in the repository. These follow strict Stylecop rules :cop:.
- [x] I have provided test coverage for my change (where applicable)

### Description

This PR updates the string representation of the pixel formats and color spaces to match the constructor and align closer to the CSS color module standard.

REF: https://www.w3.org/TR/css-color-4/

### CODE REVIEW

**Comment by iamcarbon** [comment]
Yep -- will add test coverage!

File: src/ImageSharp/ColorSpaces/CieXyChromaticityCoordinates.cs [hunk 1]
```diff
@@ -2,22 +2,21 @@
...
-    internal readonly struct CieXyChromaticityCoordinates : IEquatable<CieXyChromaticityCoordinates>, IAlmostEquatable<CieXyChromaticityCoordinates, float>
+    [StructLayout(LayoutKind.Sequential)]
+    internal readonly struct CieXyChromaticityCoordinates
+        : IEquatable<CieXyChromaticityCoordinates>, IAlmostEquatable<CieXyChromaticityCoordinates, float>
     {
-        /// <summary>
-        /// The backing vector for SIMD support.
-        /// </summary>
-        private readonly Vector2 backingVector;
+        // NOTE: We don't implement a backing vector on this class to avoid runtime bugs.
```
Comment (JimBobSquarePants) [inline src/ImageSharp/ColorSpaces/CieXyChromaticityCoordinates.cs]: Way ahead of you 😉 

https://github.com/SixLabors/ImageSharp/commit/1223f786240d05cd7c2abd80565389a7b747dd1e#diff-fdada74c86b339c6ed2bc530c7c8b390R13

Comment (iamcarbon) [reply src/ImageSharp/ColorSpaces/CieXyChromaticityCoordinates.cs]: Hah! I was going remove the backing vector from the other colorspaces next! Nice to know that we can eek even more performance out of this! 

Are we going to try to get your color-transforms branch back in sync or should I carry forward on these refactorings (referencing your branch) ?

Comment (iamcarbon) [reply src/ImageSharp/ColorSpaces/CieXyChromaticityCoordinates.cs]: Reimplemented. Was there anything else other than clean up & bulk conversions in this branch we want to bring over?

**Comment by iamcarbon** [comment]
Ready for review.

@JimBobSquarePants  -- let me know when you're branch PR is up. I want to take another pass on this after that's merged in.

File: tests/ImageSharp.Tests/Colorspaces/StringRepresentationTests.cs [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            { new LinearRgb(random), "LinearRgb(1,1,1)" },  // clamping to 1 is expected
+            { new Rgb(random), "Rgb(1,1,1)" },              // clamping to 1 is expected
+            { new Hsl(random), "Hsl(42.4,1,1)" },           // clamping to 1 is expected
+            { new Hsv(random), "Hsv(42.4,1,1)" },           // clamping to 1 is expected
+            { new YCbCr(random), "YCbCr(42.4,94.5,83.4)" },
+       };
+
+        [Theory]
+        [MemberData(nameof(TestData))]
+        public void StringRepresentationsAreCorrect(object color, string text)
```
Comment (antonfirsov) [inline tests/ImageSharp.Tests/Colorspaces/StringRepresentationTests.cs]: 👍 nice job!

**Comment by iamcarbon** [comment]
@JimBobSquarePants -- going to rebase and resubmit this PR when you're done with your stuff.

**Comment by JimBobSquarePants** [comment]
Closing as changes were added as part of #664

**Pull request closed without merging**